packages/combo-box/test/overlay-position.test.js

```diff
@@ -213,78 +213,4 @@ describe('overlay position', () => {
       });
     });
   });
-
-  describe('overlay resizing', () => {
-    const minHeight = 116;
```

Comment:
These tests were added in https://github.com/vaadin/vaadin-combo-box/pull/151 and rely on hardcoded `116px` min height that was used back then. Related logic was removed in #2497. Also, we have new tests added in #5432.